### PR TITLE
Add two led-display related entries. Remove toaster because it is fake.

### PR DESCRIPTION
**Pull request by zoli90**

### CODE REVIEW

File: storage/db/en/db.json [hunk 1]
```diff
@@ -67,21 +67,6 @@
       "tags": ["device", "nDoom", "calculator", "baremetal"],
       "previewImgUrl": "https://github.com/zvolchak/can-it-run-doom/blob/main/storage/images/doom-on-calculator.png?raw=true"
     },
-    {
-      "id": "1005",
-      "title": "A Toaster",
```
Comment (zvolchak) [inline storage/db/en/db.json:72]: haha! yes it is indeed fake!

**Review by zvolchak — APPROVED** [review]
Looks good! Thank you!

File: webapp/public/db/en/db.json [hunk 2]
```diff
@@ -765,9 +750,40 @@
...
+    },
+    {
+      "id": "1045",
+      "title": "Doom on a building",
+      "description": "Playing Doom E1M1 on the Schönherz Mátrix - an annual event of showcasing mostly animations on a building. Doom is running on GZDoom and streamed through HDMI, but we only used the Mátrix as a display. It has a size of 22.5 x 28.5 meters, with a resolution of 26 x 32 pixels.",
+      "authors": [{
+        "name": "Zoltán Bugir",
+        "url": "https://github.com/zoli90/"
+      }],
+      "sourcesUrl": [{ "name": "YouTube", "url": "https://www.youtube.com/live/eBSH0kA1RNQ&t=5287"}],
```
Comment (zvolchak) [inline webapp/public/db/en/db.json:779]: On which minute is Doom in that video?

File: webapp/public/db/en/db.json [hunk 3]
```diff
@@ -765,9 +750,40 @@
...
+      "sourceCodeUrl": [
+        { "name": "Mod and settings", "url": "https://github.com/zoli90/schonherz-doom"}
+      ],
+      "isFirstLevelComplete": true,
+      "publishDate": "Sept 29, 2024",
+      "tags": ["GZDoom", "led-display"],
+      "previewImgUrl": "https://raw.githubusercontent.com/zvolchak/can-it-run-doom/main/storage/images/doom-on-a-building.png"
```
Comment (zvolchak) [inline webapp/public/db/en/db.json:786]: if you can - please re-scale it to 400x400 pixels (don't have to me exactly that number, just something close to it). It just helps with keeping the loading time and resource usage lower. It's okay if you can't re-scale it at this moment.

File: webapp/public/db/en/db.json (hunk 2 above)
Comment (zoli90) [reply webapp/public/db/en/db.json]: In the meantime I got feedback from other people too that https://www.youtube.com/live/eBSH0kA1RNQ?si=mYQuPbXxfGQ_L6og&t=5287 does not jump to 1:28:08 automatically. I replaced it with https://www.youtube.com/watch?v=eBSH0kA1RNQ&t=5287

**Pull request merged**